src/tests/countryReducer.test.js

```diff
@@ -0,0 +1,107 @@
+import '@testing-library/jest-dom';
+import { GET_COUNTRY, GET_REGION, GET_SUBREGION } from '../redux/constants';
+import countryReducer from '../redux/home/home';
+
+describe('Test purity of countryReducer', () => {
+  it('Should return the initial state', () => {
+    const initialState = [];
+    const action = { type: 'UNKNOWN' };
+    expect(countryReducer(initialState, action)).toEqual(initialState);
+  });
```

Comment:
- [ ] Great job so far 👏🏻  However I recommended you add more unit tests for several components of the app as requested on the project requirements. ✍🏻